feedback.md

```diff
@@ -0,0 +1,29 @@
+# INFO 343 Final Project
+
+August Carow, Katie Goulding, Amanda Shen, Matthew Farmer 
+
+<!-- Description of what we will build -->
+
+## Introduction:
+Our team is going to create a web application. The most basic function of the web app will be a timer that you can use to time the length of your shower. More importantly, the shower timer will estimate how much water you used and how much it costs. If you take an especially long shower, the app might prompt you with a depressing environmental fact. The app may also have functionality that allows you to see the difference between you and your friend’s water usage. The greater goal of this simple app is to encourage people to take shorter showers through a few different strategies, thus conserving water and energy.
+
```

Comment:
Are you going to have user profiles? Will I be able to track my shower usage over time?
What about comparing my usage versus my anonymous neighbors? Do I shower the longest or shortest within 1-mile? 5 miles? 10 miles? etc.

Reply:
A timer app won't be enough - be sure to have enough complexity so that you have something cool to show off.